code/Base/GamemodeSystem/Entities/GamemodeEntity.cs

```diff
@@ -0,0 +1,27 @@
+namespace Sports;
+
+public partial class GamemodeEntity : Entity, IGamemodeEntity
```

Comment:
Add a nice comment above here saying what it does - I see people getting confused with BaseGamemode and this entity.